crt_portal/cts_forms/templates/forms/report_date.html

```diff
@@ -1,56 +1,14 @@
 {% extends "forms/report_base.html" %}
 
 {% block form_questions %}
-  <div class="crt-portal-card">
-    <div class="crt-portal-card__content crt-portal-card__content--lg">
-      {{ block.super }}
-      <fieldset class="usa-fieldset">
-        <legend class="em-text ">
-          {{ wizard.form.date_question }}
-        </legend>
+<div class="crt-portal-card">
+  <div class="crt-portal-card__content crt-portal-card__content--lg">
+    {{ block.super }}
+
+    <h2>{{ current_step_title }}</h2>
+
+    {% include "forms/question_cards/date.html" with question=wizard.form.date_question month=wizard.form.last_incident_month day=wizard.form.last_incident_day year=wizard.form.last_incident_year help_text=wizard.form.help_text %}
```

Comment:
`wizard.form` and `form` are equivalent in the context here, good spot to trim a bit and reference `form` directly

e.g.
```python
    {% include "forms/question_cards/date.html" with question=form.date_question month=form.last_incident_month day=form.last_incident_day year=form.last_incident_year help_text=form.help_text %}
```